libraries/testing/tester.cpp

```diff
@@ -20,6 +20,32 @@ eosio::chain::asset core_from_string(const std::string& s) {
 }
 
 namespace eosio { namespace testing {
+
+   std::ostream& operator<<(std::ostream& os, setup_policy p) {
+      switch(p) {
+         case setup_policy::none:
+            os << "none";
+            break;
+         case setup_policy::old_bios_only:
+            os << "old_bios_only";
+            break;
+         case setup_policy::preactivate_feature_only:
+            os << "preactivate_feature_only";
+            break;
+         case setup_policy::preactivate_feature_and_new_bios:
+            os << "preactivate_feature_and_new_bios";
+            break;
+         case setup_policy::old_wasm_parser:
+            os << "old_wasm_parser";
+            break;
+         case setup_policy::full:
+            os << "full";
+            break;
```

Comment:
Nice to have a default here, though it should never happen.

Reply:
You do get a `warning` if a new value is added, at least with `clang`:
```
tester.cpp:25:14: warning: enumeration value 'next' not handled in switch [-Wswitch]
      switch(p) {
```

Added a `default`.